<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VGRadarPPIViewer</class>
 <widget class="QWidget" name="VGRadarPPIViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VGRadarPPIViewer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>加载数据</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>范围</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxRange">
       <property name="suffix">
        <string> 公里</string>
       </property>
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>99999999.000000000000000</double>
       </property>
       <property name="value">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>扫描周期</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxT">
       <property name="suffix">
        <string> 秒</string>
       </property>
       <property name="minimum">
        <double>0.010000000000000</double>
       </property>
       <property name="maximum">
        <double>999999.000000000000000</double>
       </property>
       <property name="value">
        <double>60.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>初始方位</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxTheta">
       <property name="suffix">
        <string> 度</string>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="VGRadarPPI" name="widget" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VGRadarPPI</class>
   <extends>QWidget</extends>
   <header>vgradarppi.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
